### DISCUSSION TITLE
Is there any documentation explaining all the settings and options in the menu???

### DISCUSSION THREAD

**Discussion by PhilBaz** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1872] [Q&A]
Is there any documentation explaining all the settings and options in the menu for the TFT.

For instance the difference between the home offset and probe offset menus.  Of all of the machine options? What is an ACK terminal notification?

Cant find any documentation...

**Comment by radek8** [comment]
Unfortunately, there is no such manual.
It can be assumed that the implementation of the display will be solved by the user partially familiar with the control of the printer.
You do not want to take on creating such a manual and help other users ;-)

**Comment by radek8** [comment]
If you enable ACK on the terminal, you will see all printer and display communication. if you disable it, you will only see the answers to your questions.
Probe offset - offset of the probe compared to the nozzle, but you know it from Marlin ...
